### DISCUSSION TITLE
Standup (Wednesday, November 9th)

### DISCUSSION THREAD

**Discussion by DavBot02** [BBGTC/food-bank-app#57] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on  EventCard, I'm finishing styling right now

**What are your priorities for the next 24 hours?**:
 -  Finished EventCard
 - Implement carousel

**Blockers**:
 - None

**Shoutouts**:
 - @CarlosCRG19  for taking care of @antonieto 's headphones

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Sent PR For linter

**What are your priorities for the next 24 hours?**:
 - Finish project architecture based on Login
 - Finish the pending assignments

**Blockers**:
 - Other Ken homework

**Shoutouts**:
 - @antonieto for getting over the pain of loosing his headphones

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished #40 

**What are your priorities for the next 24 hours?**:
 - Write tests for #40 
 - Implement admin account logic for the donation verification feature

**Blockers**:
 - Got some elective homework

**Shoutouts**:
 - @BrandonMagana for cracking the Google interview

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on donation category cards
 - Read about navigation types (looks hard)

**What are your priorities for the next 24 hours?**:
 - Finish logic for donation

**Blockers**:
 - Not sure how to implement logic
 - Navigation types are weird 

**Shoutouts**:
 - @CarlosCRG19 for being a capable thief
